v0/destinations/webhook/transform.js

```diff
@@ -1,39 +1,51 @@
-const { defaultPostRequestConfig, defaultRequestConfig } = require("../util");
+const {
+  defaultPostRequestConfig,
+  defaultGetRequestConfig,
+  defaultRequestConfig,
+  getHashFromArray,
+  getFieldValueFromMessage,
+  flattenJson
+} = require("../util");
+const { EventType } = require("../../../constants");
 
-function handleEvent(event) {
-  const { message, destination } = event;
-  const response = defaultRequestConfig();
-  if (destination.Config.webhookUrl) {
-    response.endpoint = destination.Config.webhookUrl;
-    response.method = defaultPostRequestConfig.requestMethod;
-    // add content-type by default as our payload is json
-    response.headers = {
-      "Content-Type": "application/json"
-    };
-
-    const configHeaders = destination.Config.headers;
-    if (configHeaders) {
-      configHeaders.forEach(header => {
-        response.headers[header.from] = header.to;
-      });
-    }
-
-    response.userId = message.userId || message.anonymousId;
-    response.body.JSON = { ...message };
-    return response;
+const getPropertyParams = message => {
+  if (message.type === EventType.IDENTIFY) {
+    return flattenJson(getFieldValueFromMessage(message, "traits"));
   }
-  throw new Error("Invalid Url in destination");
-}
+  return flattenJson(message.properties);
+};
 
-const process = event => {
+function process(event) {
   try {
```

Comment:
Do we need the `try-catch` block here?

Reply:
Since we have added some logic now(earlier it was very simple), so i guess its better to have a try catch